source/ast/ASTContext.cpp

```diff
@@ -136,7 +136,8 @@ void ASTContext::setAttributes(const Expression& expr,
 
 void ASTContext::addDriver(const ValueSymbol& symbol, const Expression& longestStaticPrefix,
                            bitmask<AssignFlags> assignFlags) const {
-    if (flags.has(ASTFlags::NotADriver) || scope->isUninstantiated())
+    if ((flags.has(ASTFlags::NotADriver) && !assignFlags.has(AssignFlags::NetAlias)) ||
```

Comment:
I think you can get rid of this change if you simply don't use the NotADriver flag when binding the net alias expressions

Reply:
fixed